### PR TITLE
Dumping state machines to streams

### PR DESCRIPTION
**Pull request by bernhard-thiele**
@sjoelund can you please review the commit?

### CODE REVIEW

File: Compiler/FrontEnd/DAEDump.mo [hunk 1]
```diff
@@ -2773,6 +2777,54 @@ algorithm
...
+          SOME(SCode.COMMENT(comment=SOME(cstr))) := comment;
+          cstr := " \"" + cstr + "\"";
+        else
+          cstr := "";
+        end try;
+        SPLIT_ELEMENTS(v,ie,ia,e,a,co,_,_,sm) := spltElems;
+
+        str := IOStream.append(str, name + cstr + "\n");
+
+	      str := dumpCompWithSplitElementsStream(sm, str);
```
Comment (sjoelund) [inline Compiler/FrontEnd/DAEDump.mo]: What is this indentation doing? Some tab-stops?

**Comment by sjoelund** [comment]
I started a Hudson non-PUSH to see if there are any actual changes to the dumping. If it does not, you probably need to add a test-case for this.

File: Compiler/FrontEnd/DAEDump.mo (hunk 1 above)
Comment (bernhard-thiele) [inline Compiler/FrontEnd/DAEDump.mo]: Yes. Doesn't show up indented in my editor. Should fix that. Anything else?

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2016-02-16_15-57-48](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/1595/console).

File: Compiler/FrontEnd/DAEDump.mo (hunk 1 above)
Comment (sjoelund) [inline Compiler/FrontEnd/DAEDump.mo]: No. The rest looks good.

Comment (bernhard-thiele) [inline Compiler/FrontEnd/DAEDump.mo]: Thanks.

**Comment by sjoelund** [comment]
Any reason this is not merged yet?

**Comment by bernhard-thiele** [comment]
I already made the changes and merged it, but this got somehow detached to this issue. The issue can be closed.

**Pull request closed without merging**